frameworks/helloworld/tests/test_placement.py

```diff
@@ -445,6 +362,116 @@ def test_updated_placement_constraints_replaced_tasks_do_move():
     assert get_task_host('hello-0-server') == other_agent
 
 
+@pytest.mark.dcos_min_version('1.12')
+@pytest.mark.sanity
+def test_scheduler_task_placement_by_marathon():
```

Comment:
This is the only new method. Rest of the diff in this file is restructuring (I reordered methods so that test_* comes first followed by helpers).